CMakeLists.txt

```diff
@@ -1,10 +1,17 @@
+
+if(POLICY CMP0091)
+  cmake_policy(SET CMP0091 NEW)
+endif()
+
 cmake_minimum_required(VERSION 3.19)
 
 project(SwiftTSC LANGUAGES C Swift)
 
 set(CMAKE_Swift_LANGUAGE_VERSION 5)
 set(CMAKE_Swift_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/swift)
+set(CMAKE_Swift_COMPILE_OPTIONS_MSVC_RUNTIME_LIBRARY MultiThreadedDLL)
 
+set(CMAKE_MSVC_RUNTIME_LIBRARY MultiThreadedDLL)
```

Comment:
Why is this `MultiThreadedDLL` and not `MultiThreaded$<$<CONFIG:Debug>:Debug>DLL`?
Don't we want it to expand to the library that matches, or are we not using the debug DLL in either case?

Same for the above.

Reply:
ah, okay, so just tying the C portions to release builds so it's just a {static,dynamic}{debug,release}Swift x release{static,dynamic}C/C++ runtime bits. okay.

Reply:
Do we need to mix the release and debug Swift with release and debug C/C++? Wouldn't it make more sense to have everything be debug or everything be release? And would it make sense to limit the debug builds to either just dll's/static-linked? Then we only need to build the debug half for both either statically or dynamically and it's no longer a combinatorial explosion. Then the debug build is only one configuration and each thing gets built three times.

Reply:
I think that if we want to reduce it further we should continue to not support static linking.  We could do dynamic only with debug and release mode and match Swift and C/C++.